src/lib/scanner.ts

```diff
@@ -21,13 +30,15 @@ export class ScanRunner {
      */
     public async run(targets: string[]): Promise<RuleResult[]> {
         // Create the arg array.
-        const args: string[] = await this.createPathlessArgArray(targets);
+        const args: Inputs = this.createPathlessArgArray(targets);
 
         // Invoke the scanner.
-        const executionResult: ExecutionResult = await this.invokeAnalyzer(args);
+        // const executionResult = await this.invokeAnalyzer(args);
```

Comment:
is there a reason why there is commented out code still?

Reply:
This can be removed. I've removed it now.